### ISSUE TITLE
Unable to locate the city in the IANA database bug

### ISSUE THREAD

**Issue by vojtechmusilek** [rick-jennings/phable#7]
I don't know if I'm doing something wrong but following read calls throw error 
`phable.parsers.json.IanaCityNotFoundError: Unable to locate the city Prague in the IANA database`

```py
with Client(uri, username, password) as client:
  points = client.eval('readAll(point and tz=="Prague" and (unit == "kW" or unit == "°C"), {limit:2})')
  points = client.read('point and tz=="Phoenix" and (unit == "kW" or unit == "°F")', 2)
  points = client.read('point and tz=="Prague" and (unit == "kW" or unit == "°C")', 2)
```

Tried Phoenix and Prague and both throw error, also tried eval and read but it is the same.
The points in skyspark are nothing special, they look valid.
Also tried running the readAll in skyspark and it returns the points as expected.

Can you replicate this issue?
(on latest 0.1.13)

**Comment by vojtechmusilek** [comment]
It was because my new enviroment was missing tzdata package, which comes in as dependency of pandas but this enviroment did not have pandas. Maybe tzdata should be required dependency of phable?

**Comment by rick-jennings** [comment]
I was not able to replicate the issue using the example code provided and Python 3.11.

I see that Phable maps Haystack's city name "Prague" to "Europe/Prague" and "Phoenix" to "America/Phoenix".  Is this what you would expect?

Here is example code:
```
from phable.parsers.json import _haystack_to_iana_tz

print(_haystack_to_iana_tz("Prague"))
print(_haystack_to_iana_tz("Phoenix"))
```

Please note that Phable relies on the `zoneinfo` module which was introduced in Python 3.11.

Which version of Python are you using?

**Comment by vojtechmusilek** [comment]
I'm using python 3.11.2. I was able to make it work after installing the tzdata package. Can you check if you have this package installed? Does it work for you when you uninstall it?

**Comment by rick-jennings** [comment]
I just ran another test using Python 3.11.2 without `tzdata` and it worked for me.

According to the Python docs in the link below:
"By default, [zoneinfo](https://docs.python.org/3.11/library/zoneinfo.html#module-zoneinfo) uses the system’s time zone data if available; if no system time zone data is available, the library will fall back to using the first-party [tzdata](https://pypi.org/project/tzdata/) package available on PyPI."

https://docs.python.org/3.11/library/zoneinfo.html#module-zoneinfo

I suspect on my computer time zone data is available and on your computer it is not.  This is why I think you see the error and I don't.

For now I would recommend you install `phable` with the `pandas` dependency.  

I think there are several possible solutions for us to explore:
1. Make `tzdata` a mandatory dependency to `phable`
2. Make `tzdata` an optional dependency to `phable`
3. Keep the existing design and update the docs

In either case I think we will update the `phable` docs.

What do you think is the best solution considering other users of `phable` may not want the `tzdata` dependency?

**Comment by vojtechmusilek** [comment]
You are right, without tzdata running this I get empty set.
```py
import zoneinfo
zoneinfo.available_timezones()
```

After installing tzdata it shows all the timezones.

I would probably go for 3 - keep it as is, update docs, and if possible add some helpful message at the end of current one like:
`Unable to locate the city Prague in the IANA database. Consider installing tzdata package.`

**Comment by rick-jennings** [comment]
Hi Vojtech, thanks again for the feedback.  I implemented your latest advice in Phable version 0.1.14.
